src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -28,6 +28,40 @@
 
 using namespace oboe;
 
+static SLuint32 OpenSLES_convertOutputUsage(Usage oboeUsage) {
+    SLuint32 openslStream = SL_ANDROID_STREAM_MEDIA;
+    switch(oboeUsage) {
+        case Usage::Media:
+            openslStream = SL_ANDROID_STREAM_MEDIA;
+            break;
+        case Usage::VoiceCommunication:
+        case Usage::VoiceCommunicationSignalling:
+            openslStream = SL_ANDROID_STREAM_VOICE;
+            break;
+        case Usage::Alarm:
+            openslStream = SL_ANDROID_STREAM_ALARM;
+            break;
+        case Usage::Notification:
+        case Usage::NotificationRingtone:
+        case Usage::NotificationEvent:
+            openslStream = SL_ANDROID_STREAM_NOTIFICATION;
+            break;
+        case Usage::AssistanceAccessibility:
+        case Usage::AssistanceNavigationGuidance:
+        case Usage::AssistanceSonification:
+            openslStream = SL_ANDROID_STREAM_SYSTEM;
+            break;
+        case Usage::Game:
+            openslStream = SL_ANDROID_RECORDING_PRESET_VOICE_COMMUNICATION;
```

Comment:
SL_ANDROID_RECORDING_PRESET_VOICE_COMMUNICATION is not a streamType.
No ideal match is available.
Maybe should be SL_ANDROID_STREAM_MEDIA.

Reply:
Oh, right, my bad. Agreed on SL_ANDROID_STREAM_MEDIA.